Description: Remove static btrfs space_cache settings 
  Calamares sets an option "space_cache", which breaks with newer kernels,
  where it should be set to "space_cache=v2". Since "space_cache=v2" is
  also default now, this hardcoded option can just be removed, which this
  patch does.
Author: Jonathan Carter <jcc@debian.org>
Bug-Debian: https://bugs.debian.org/1080158
Last-Update: 2024-09-03

--- lingmo-installer-config-12.0.9.orig/calamares/modules/fstab.conf
+++ lingmo-installer-config-12.0.9/calamares/modules/fstab.conf
@@ -1,7 +1,7 @@
 ---
 mountOptions:
     default: defaults,noatime
-    btrfs: defaults,noatime,space_cache,autodefrag
+    btrfs: defaults,noatime,autodefrag
 ssdExtraMountOptions:
     ext4: discard
     jfs: discard
